api/v2/performanceprofile_types.go

```diff
@@ -123,6 +126,36 @@ type NUMA struct {
 	TopologyPolicy *string `json:"topologyPolicy,omitempty"`
 }
 
+// Net defines a set of network related features
+type Net struct {
+	// UserLevelNetworking defines if the specified network devices . Defaults to "false".
+	UserLevelNetworking *bool `json:"userLevelNetworking,omitempty"`
+	// Devices contains a list of network device representations that will be
+	// set with a netqueue count equals to CPU.Reserved .
+	// If no devices are specified then the default is all devices.
+	Devices []Device `json:"devices,omitempty"`
+}
+
+// Device defines a way to represent a network device in several options:
+// device name, vendor ID, model ID, PCI path and MAC address
+type Device struct {
+	// Network device name
+	// +optional
+	Name *string `json:"name,omitempty"`
+	// Network device vendor ID represnted as a 16 bit Hexmadecimal number.
+	// +optional
+	VendorID *string `json:"vendor,omitempty"`
+	// Network device model ID (device) represnted as a 16 bit hexmadecimal number.
+	// +optional
+	ModelID *string `json:"model,omitempty"`
```

Comment:
Here its a bit more tricky. mentioning only model probably does not make much sense without specifying vendor ID but this is still acceptable in terms of tuned network plugin

Reply:
fine. Point is: shall we pass through the network plugin? (IOW: should we expose all the options the network plugin allows _in the same format_ the network plugin allows?) This is something we need to carefully evaluate because adding options to the API seems safe and easy, while removing them is painful.

For example: can we start allowing a `vendorid:modelid` pair supporting a simple wildcard mechanism to specfy only `vendorid` ?

Reply:
it could look like:

- `vendorid:modelid` - match this exact pair
- `vendorid` - match all models from this vendor

IOW:

1. if `:` is in the string, treat like a pair
2. otherwise, treat it like a `vendorid`

Reply:
Please see my comment above, anyway please use deviceid as is the standard naming.